### PR TITLE
README.md Markdown link typo fix

### PR DESCRIPTION
**Pull request by thenaturalist**
This commit fixes a Markdown link typo and includes link checking for references.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,8 +207,10 @@ Copyright &copy; 2020 [Mineiros GmbH][homepage]
 [badge-terraform]: https://img.shields.io/badge/terraform-0.13%20and%200.12.20+-623CE4.svg?logo=terraform
 [badge-slack]: https://img.shields.io/badge/slack-@mineiros--community-f32752.svg?logo=slack
 
+<!-- markdown-link-check-disable -->
 [build-status]: https://mineiros.semaphoreci.com/projects/terraform-github-team
```
Comment (soerenmartius) [inline README.md:211]: the project is public in semaphore so we can indeed have it checked by markdown-link-checker.

File: README.md [hunk 2]
```diff
@@ -207,8 +207,10 @@ Copyright &copy; 2020 [Mineiros GmbH][homepage]
...
 [releases-github]: https://github.com/mineiros-io/terraform-github-team/releases
```
Comment (soerenmartius) [inline README.md:212]: same here, the repository is public already

File: README.md [hunk 3]
```diff
@@ -217,12 +219,13 @@ Copyright &copy; 2020 [Mineiros GmbH][homepage]
...
-
+<!-- markdown-link-check-disable -->
 [variables.tf]: https://github.com/mineiros-io/terraform-github-team/blob/master/variables.tf
 [examples/]: https://github.com/mineiros-io/terraform-github-team/tree/master/examples
 [Issues]: https://github.com/mineiros-io/terraform-github-team/issues
 [LICENSE]: https://github.com/mineiros-io/terraform-github-team/blob/master/LICENSE
 [Makefile]: https://github.com/mineiros-io/terraform-github-team/blob/master/Makefile
 [Pull Requests]: https://github.com/mineiros-io/terraform-github-team/pulls
 [Contribution Guidelines]: https://github.com/mineiros-io/terraform-github-team/blob/master/CONTRIBUTING.md
+[Github teams]: (https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/organizing-members-into-teams)
```
Comment (soerenmartius) [inline README.md:230]: these links are all public, so remove the mardown-link-check. 

To test locally after removing you can just run `make test/pre-commit`

Comment (mariux) [reply README.md:230]: The () need to be removed

Comment (soerenmartius) [reply README.md:230]: good catch :D

Comment (soerenmartius) [reply README.md:230]: 🦅

**Comment by mariux** [comment]
github provider 2.9.0 seems to be broken as token is required now...

**Comment by soerenmartius** [comment]
> github provider 2.9.0 seems to be broken as token is required now...

The token was always required by the provider? It doesn't make much sense without the token anyways, does it ?

But yeah, pre-commit fails because of the missing token: https://mineiros.semaphoreci.com/jobs/9157e276-7ff7-4e3f-abbe-86eb16549329

**Comment by mariux** [comment]
https://github.com/terraform-providers/terraform-provider-github/pull/464#issuecomment-651187596

**Comment by mariux** [comment]
we specify the token vie environment so this was fine before..

**Comment by soerenmartius** [comment]
> we specify the token vie environment so this was fine before..

Yeah, we are providing it through the environment variable `GITHUB_TOKEN`

**Comment by mariux** [comment]
> The token was always required by the provider? It doesn't make much sense without the token anyways, does it ?

See https://github.com/terraform-providers/terraform-provider-github/pull/464#issuecomment-651254293 

As the provider was refactored to not only support organizations, the token went back to the required state. Setting the `GITHUB_TOKEN` environment variable to any string will fix `terraform validate`. tested and confirmed it with 2.8.0 and 2.9.0 of the provider.

We should also check out what this refactoring means for the features of our GitHub modules.. maybe we can add some awesome stuff...maybe we can't ;) 

The fix you applied for providing the variable needs to be integrated into the other two repositories. I can create PRs for this. thanks.

**Comment by mariux** [comment]
I raised the issue upstream: https://github.com/terraform-providers/terraform-provider-github/issues/503 
don't think we need further steps on our side as this is fixed now by providing the token. 
It still breaks local pre-commits when no token is set, but it can be set to a random value.

**Pull request merged**